py3status/modules/stopwatch.py

```diff
@@ -0,0 +1,140 @@
+# -*- coding: utf-8 -*-
+"""
+A simple stopwatch.
+
+This is a very basic stopwatch. You can start, pause and reset the
+stopwatch.
+
+Button 1 starts/pauses the stopwatch.
+Button 2 resets stopwatch.
```

Comment:
Kill the comments. The (missing) docs will take care of this.

Reply:
Done